### PR TITLE
ENH: Add MEG sensor option to coreg

### PR DESCRIPTION
**Pull request by larsoner**
Allows plotting the MEG sensors in addition to the helmet, and also should fix the rendering bugs you were seeing during the sprint @wmvanvliet by introducing a 1mm offset between the MEGIN sensor geometries which otherwise are perfectly coincident (which is bad news when using translucent geometry, at least when using depth peeling which is otherwise generally a good idea):
```
mne coreg -s sample --subjects-dir=~/mne_data/MNE-sample-data/subjects -f ~/mne_data/MNE-sample-data/MEG/sample/sample_audvis_raw.fif
```
image

As an aside @wmvanvliet I noticed something when debugging the rendering issue -- if you want your rendering to be (closer to) as smooth as MNE-C you can disable depth peeling and it's *much* faster. But it has the side effect of messing up translucent geometry.

### CODE REVIEW

**Comment by larsoner** [comment]
@jasmainak could you look? I think you requested this in #11276

**Comment by drammock** [comment]
I'm seeing the sensors disappear / replaced by a greyish square when I zoom in. Is that expected?

[coreg.webm](https://github.com/mne-tools/mne-python/assets/1810515/d1d35c77-ede8-4197-8ed4-9f0c4a2f2995)

File: mne/gui/_coreg.py [hunk 1]
```diff
@@ -102,6 +102,10 @@ class CoregistrationUI(HasTraits):
         If True, display the head shape points. Defaults to True.
     eeg_channels : bool
         If True, display the EEG channels. Defaults to True.
+    meg_channels : bool
+        If True, display the MEG channels. Defaults to False.
+    fnirs_channels : bool
+        If True, display the MEG channels. Defaults to True.
```
Comment (drammock) [inline mne/gui/_coreg.py:108]:
```suggestion
    fnirs_channels : bool
        If True, display the fNIRS channels. Defaults to True.
```

**Review by drammock — APPROVED** [review]
just one typo

**Comment by larsoner** [comment]
Hmm... could be the depth peeling failing or the view frustum doing something odd. But more likely it's "just" a rendering bug of some sort for your GPU and/or version of VTK, on macOS I don't see it:

Screenshot 2023-10-11 at 4 15 28 PM

I'll commit the typo fix and mark for merge when green!

**Pull request merged**